### PR TITLE
ENH : Implement a function for computing the DVARS

### PR DESCRIPTION
**Pull request by celprov**

### CODE REVIEW

**Review by effigies — APPROVED** [review]
Looks good!

File: findoutlie/metrics.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    # In [3]: np.mean(arr, axis=1)
+    # Out[2]: array([3., 6.])
+    #
+    # You may be be able to solve this in four lines, without a loop.
+    # But solve it any way you can.
+    data = img.get_fdata()
+    data = np.reshape(data, (-1, data.shape[-1]))
+    data_shift = np.roll(data, 1, axis=-1)
+    diff =  data_shift - data
+    diff = diff[...,1:]
```
Comment (hugofluhr) [inline findoutlie/metrics.py]:
```suggestion
    voxel_per_time = np.reshape(data, (-1, data.shape[-1]))
    diff = np.diff(voxel_per_time)
```

**Review by hugofluhr — CHANGES_REQUESTED** [review]
used np.diff to be more concise

**Review by celprov — COMMENTED** [review]
This shortcut to compute differences of an array sounds reasonable to me.

**Review by hugofluhr — APPROVED** [review]

**Pull request merged**